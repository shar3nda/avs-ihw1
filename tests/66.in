10
4 7 6 10 0 3 7 4 1 1