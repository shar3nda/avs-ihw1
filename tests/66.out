-3 8 -3 7 -6 3 5 -6